src/components/action-popover/action-popover.spec.tsx

```diff
@@ -806,6 +818,7 @@ describe("ActionPopover", () => {
         const { items } = getElements();
 
         act(() => {
+          items.at(1).simulate("mouseenter");
```

Comment:
Why are these events being simulated twice now?

Reply:
Just for the sake of having test coverage of these lines `if (mouseEnterTimer.current) clearTimeout(mouseEnterTimer.current);` and `if (mouseLeaveTimer.current) clearTimeout(mouseLeaveTimer.current);` in `action-popover-item.component.tsx`

Reply:
I'll simply add another test to cover these if statements properly

Reply:
Ah good point I hadn't considered that you'd needed to add extra guards